### DISCUSSION TITLE
content bar highlighted title does not match actual selection

### DISCUSSION THREAD

**Discussion by lucasrodes** [pradyunsg/furo#376] [Potential Issue] [answered]
I am unsure if this is a furo-related issue or not, so apologize in advance if not.

### Context

My setup is with sphinx and readthedocs.org, using furo theme. Here is my [conf.py](https://github.com/owid/covid-19-data/blob/master/scripts/docs/conf.py) and all the [documentation folder](https://github.com/owid/covid-19-data/blob/master/scripts/docs) . The website is live at [owidcovid-19-data.rtfd.io](http://owidcovid-19-data.rtfd.io/).

### Problem
When in the [live version](http://owidcovid-19-data.rtfd.io), which is deployed on readthedocs.org, the content bar doesn't seem to highlight the correct part of the document.

Take the example from page http://owidcovid-19-data.rtfd.io/en/latest/data-pipeline.html. Note in the following screen recording how the content bar titles behave (it doesn't match the selection):

![furo](https://user-images.githubusercontent.com/18101289/154165258-870515bd-bbbb-4b8e-a786-18202d9bae3b.gif)
 

It does work locally (serving the files via `python -m http.server`).

**Comment by pradyunsg** [comment, marked as answer]
Made into #377. This is a bug. :)

**Reply by lucasrodes** [reply]
Thanks, @pradyunsg!
